.travis.yml

```diff
@@ -193,6 +200,7 @@ install:
             sed -i -e 's/40900/40800/g' lib/librte_eal/common/include/arch/arm/rte_vect.h
             sed -i -e 's/!(/!(defined(__arm__) \&\& defined(__clang__) || /g' lib/librte_eal/common/include/arch/arm/rte_byteorder.h
             sed -i -e 's/__GNUC__/defined(__arm__) \&\& defined(__clang__) || __GNUC__/' lib/librte_eal/common/include/generic/rte_byteorder.h
+            sed -i -e 's,\$(CC),\0 $(EXTRA_CFLAGS),g' lib/librte_acl/Makefile
```

Comment:
should this go to dpdk instead of odp?

Reply:
I know nothing about contributing to DPDK. Also what is the chance of fixing old DPDK releases (which we use)?

Reply:
if we push this to dpdk then we will not need to fix it with new releases.